.github/workflows/ci.yml

```
@@ -203,7 +203,7 @@ jobs:
 
     - name: CLI tests
```

Alternatively you could also set `continue-on-error` to true, which might make it easier to find failing jobs later, but also fine as is.
```suggestion
    - name: CLI tests
      continue-on-error: true
```